fe/fe-core/src/main/java/org/apache/doris/alter/AlterJobV2.java

```diff
@@ -61,7 +61,7 @@ public boolean isFinalState() {
     }
 
     public enum JobType {
-        ROLLUP, SCHEMA_CHANGE
+        ROLLUP, SCHEMA_CHANGE, DECOMMISSION_BACKEND
```

Comment:
`DECOMMISSION_BACKEND`, do we need this?

Reply:
Yes, for example in decommission clause, it will use the enum.